gtsam/base/DSFMap.h

```diff
@@ -120,4 +121,12 @@ class IndexPair : public std::pair<size_t,size_t> {
   inline size_t i() const { return first; };
   inline size_t j() const { return second; };
 };
+
+typedef std::vector<IndexPair> IndexPairVector;
+typedef std::set<IndexPair> IndexPairSet;
+
+inline IndexPairVector IndexPairSetAsArray(IndexPairSet& set) { return IndexPairVector(set.begin(), set.end()); }
+
+typedef std::map<IndexPair, IndexPairSet> IndexPairSetMap;
+typedef DSFMap<IndexPair> DSFMapIndexPair;
```

Comment:
`using` instead of `typedef`?
My understanding is that `using` plays nicer with templates. I am not sure if the wrapper supports this though.

Reply:
I think other such definitions use typedef, so let's keep it consistent :)